lib/RoadizCoreBundle/src/Node/NodeTranstyper.php

```diff
@@ -28,7 +28,7 @@ final class NodeTranstyper
 
     public function __construct(
         ManagerRegistry $managerRegistry,
-        private readonly NodeTypes $nodeTypesBag,
+        private readonly DecoratedNodeTypes $nodeTypesBag,
```

Comment:
NodeTranstyper is a technical service.
It should not require decorated node-types !

Reply:
This is not resolved.